curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148e41c728f65138addf9cc.md

```diff
@@ -7,14 +7,24 @@ dashedName: step-66
 
 # --description--
 
-Setting the `scroll-behavior` to `smooth` is preferred by most users. However, some users find this to be too slow, and prefer to have the scrolling happen instantaneously.
+Certain types of motion-based animations can cause discomfort for some users. In particular, people with <dfn>vestibular</dfn> disorders have sensitivity to certain motion triggers.
```

Comment:
```suggestion
Certain types of motion-based animations can cause discomfort for some users. In particular, people with <em>vestibular disorders</em> have sensitivity to certain motion triggers.
```

Reply:
I thought we used `<dfn>` when introducing a new word/concept, is that not the case here?

The reason I only included the word _vestibular_ and not disorders, is because I see disorders as a catch-all word used to cover multiple disorders. There are [multiple types](https://vestibular.org/article/diagnosis-treatment/types-of-vestibular-disorders/) of vestibular disorders. But I guess I can see "vestibular disorders" as an umbrella term and as such both words should be emphasized.

Anyway, I'm not against the suggested change just wanted to ask for clarification and maybe some feedback.

Reply:
> used <dfn> when introducing a new word/concept

We can keep using this way, but`dfn` use is in providing definition after the tag.

For example

This is our current use

> Lasjorg is great <dfn>contributor</dfn> to freeCodeCamp.

What I think `dfn` should be from reading the docs

> freeCodeCamp's <dfn>contributors</dfn> are volunteers who give their time to make freeCodeCamp better, lasjorg is great contributor to freeCodeCamp 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dfn

I remember opening an issue, where bbsmooth mentioned they were fine this way, so this is literally a extremely unimportant nitpick.